src/frontend/src/components/EditUser/index.tsx

```diff
@@ -81,10 +101,46 @@ class EditUser extends React.Component<Props, State> {
         missingPassword: this.state.password.length === 0,
         invalidPassword: this.state.password.length > 0 && this.state.password.length < 8,
         mismatchPasswords: this.state.password !== this.state.confirmPassword
+      },
+      () => {
+        if (this.state.missingName
+          || this.state.invalidEmail
+          || this.state.missingPassword
+          || this.state.invalidPassword
+          || this.state.mismatchPasswords) {
+            return
```

Comment:
We probably should just invert this if statement 

`if !(...) { this.dispatchEditUser(); }`

Reply:
I changed it to remove the else statement.